test/dynamo/test_bytecode_hook.py

```diff
@@ -12,6 +12,7 @@ def fn(a, b):
         def hook(code, out_code):
             print(code)
             print(out_code)
+            return code
```

Comment:
Add a simple test case for bytecode hook to change the output bytecode (to the original bytecode).